### ISSUE TITLE
SproutFormsAddressField - Cannot read property 'querySelectorAll' of null

### ISSUE THREAD

**Issue by engram-design** [barrelstrength/craft-sprout-forms#378]
Getting the following JS error using the basic templates after an update to the latest version.

```
Uncaught TypeError: Cannot read property 'querySelectorAll' of null
    at new SproutFormsAddressField (addressfield.js?v=1575339021:10)
```

Caused by https://github.com/barrelstrength/craft-sprout-forms/blob/v3/src/templates/_components/formtemplates/basic/form.twig#L85

Change `new SproutFormsAddressField();` to `new SproutFormsAddressField('{{ id }}');`

**Comment by engram-design** [comment]
Now I'm not so sure this is the correct fix. Fixing this throws a `Uncaught ReferenceError: SproutFormsCheckableInputs is not defined`. From what I can tell, this file is bundled in the `accessibility.js` file - should this be included in the basic template? This seems to be a new change including these JS files...

**Comment by BenParizek** [comment]
Thanks for reporting @engram-design 

Looks like we missed a couple of small details in the Basic Templates. You were on the right path, but there are a couple more edits that are needed that you can see here:
https://github.com/barrelstrength/craft-sprout-forms/commit/5de9deca2d44d62d31d8949a6392e6d7a9501bc3

We'll get these fixed in the next release which I'll have out shortly.

**Comment by engram-design** [comment]
No worries, I thought after my second comment that it'd need someone more in the know to look at it. Thanks1

**Comment by BenParizek** [comment]
@engram-design This has been addressed in v3.6.5 which is now released.
